test/busted/outputHandlers/nvim.lua

```diff
@@ -1,10 +1,13 @@
 local pretty = require 'pl.pretty'
 local global_helpers = require('test.helpers')
 
--- Colors are disabled by default. #15610
 local colors = setmetatable({}, {__index = function() return function(s) return s == nil and '' or tostring(s) end end})
 if os.getenv "TEST_COLORS" then
```

Comment:
Could default to true here if it's undefined. That avoids needing to change `cmake/RunTests.cmake`. That also keeps the semantics in one place.